frontend/packages/dev-console/src/components/helm/HelmActionForm.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FormikProps, FormikValues, useFormikContext } from 'formik';
+import { safeDump } from 'js-yaml';
+import { TextInputTypes, Grid, GridItem } from '@patternfly/react-core';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { InputField, FormFooter, FlexForm, YAMLEditorField, DropdownField } from '@console/shared';
+import FormSection from '../import/section/FormSection';
+import { HelmActionType } from './helm-types';
+
+interface HelmActionFormProps {
+  chartHasValues: boolean;
+  chartVersions?: {};
+  activeChartVersion?: string;
+  onChartVersionChange?: (value: string) => string;
```

Comment:
Don't think you this this a prop.

Reply:
same as https://github.com/openshift/console/pull/4785#discussion_r399575853